generated/gitops-template/azure/azure-pipelines.yml

```diff
@@ -3,13 +3,24 @@
 trigger:
 - main
 
+# Using self-hosted 'resourcehub' agent pool by default
+# Change accordingly when using a different agent pool
+# Can be deleted if self-hosted agents are not being used
+# See https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/agents
 pool:
   name: resourcehub
 
 container:
   image: quay.io/redhat-appstudio/rhtap-task-runner:latest
   options: --privileged
 
+# Using 'project-variables' variable group by default
+# Change accordingly when using a different variable group
+# Can be deleted if the variables are set differently
+# See https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables
+variables:
+- group: project-variables
```

Comment:
Can we call this maybe just `rhtap`? I'd rather have something that makes it clear this is needed for rhtap.

Not sure if having "variables" in the name is needed since it is already referring to a variable group entity?